modules/storebooks.js

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line import/prefer-default-export
+export class StoreBooks {
+  static getBooks() {
+    let books;
+
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = StoreBooks.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(title) {
+    const books = StoreBooks.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```

Comment:
- [ ] Your app does not really clear the local storage on removing the books. When I remove books and reload all the books still come back on the page. Kindly fix this.

  books are added | books are removed | Page is refreshed
  :---:|:---:|:---:
  ![awe-1](https://user-images.githubusercontent.com/54848363/183473928-9b74b4f2-c4f0-42eb-84c7-349466065499.jpg) | ![awe-2](https://user-images.githubusercontent.com/54848363/183473921-e6837524-f6db-4dae-a044-5a42e525ae83.jpg) | ![awe-1](https://user-images.githubusercontent.com/54848363/183473928-9b74b4f2-c4f0-42eb-84c7-349466065499.jpg)

Reply:
Thank You for your Review I have Fixed the issue.